### PR TITLE
Secondbranch

### PR DESCRIPTION
**Pull request by govardanareddy**
Succesfuully chnaged version,please check it out and if you satisfied with this yo u can merge this code.

### CODE REVIEW

File: rest/rest-template/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 	<groupId>com.therealdanvega</groupId>
 	<artifactId>rest-template</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
```
Comment (HarshaGovindubb) [inline rest/rest-template/pom.xml:8]: what the fuck is this.
For the version change your doing PR. your fired

File: rest/rest-template/pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
+	<version>0.0.3-SNAPSHOT</version>
```
Comment (HarshaGovindubb) [inline rest/rest-template/pom.xml:8]: change this immediately

**Comment by HarshaGovindubb** [comment]
@govardanareddy ohh my bad perfect code.

**Comment by HarshaGovindubb** [comment]
@govardanareddy merged.

**Comment by govardanareddy** [comment]
thank you

**Pull request merged**